amun/base/kustomization.yaml

```diff
@@ -9,6 +9,9 @@ resources:
   - argo-workflows/inspection-build-with-cpu.yaml
   - argo-workflows/inspection-run.yaml
   - argo-workflows/inspection-run-with-cpu.yaml
+  - argo-workflows/parse-inspection-inputs.yaml
+  - argo-workflows/send-message.yaml
```

Comment:
do we need this template here? I see only `send-messages.yaml` is used in inspection workflow templates